### PR TITLE
Updated help page

### PR DESCRIPTION
**Pull request by Valentine-Mario**
updated help page command with a minified version of the docs

### CODE REVIEW

**Comment by Valentine-Mario** [comment]
@arctic-hen7 please review.

**Comment by arctic-hen7** [comment]
Hey sorry for the delay! Reviewing now...

**Review by arctic-hen7 — CHANGES_REQUESTED** [review]
That looks really great, thanks so much! Just a few things with grammar and capitalisation, and I'd prefer the help page be defined in the `BONNIE_HELP_PAGE` constant, meaning only `help_page.rs` needs to be changed.

**Comment by arctic-hen7** [comment]
Please note, this is the first pull request I've ever reviewed, so please tell me if there's anything I've done horribly wrong! Also, thanks a lot for the contribution, it means a lot!

**Comment by Valentine-Mario** [comment]
> Please note, this is the first pull request I've ever reviewed, so please tell me if there's anything I've done horribly wrong! Also, thanks a lot for the contribution, it means a lot!

Lol. The review was great. I highly appreciate the feedback

File: src/help_page.rs [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+        Arguments can be parsed into a command as well:
+
+        [script]
+        greet.args = [\"firstname\",
+                     \"lastname\"]
+
+    To run the command in the terimal:
+        'bonnie greet Donald Knuth'
+        
+    Arguments can be used multiple times in a command::
```
Comment (arctic-hen7) [inline src/help_page.rs:19]: Typo: two colons rather than one.

**Review by arctic-hen7 — CHANGES_REQUESTED** [review]
Fantastic, thank you very much! Just one '::' rather than ':'!

**Review by arctic-hen7 — APPROVED** [review]
Looks great, thanks so much!

**Comment by Valentine-Mario** [comment]
> Looks great, thanks so much!

I'll suggest you add a changelog.md file so that task can be outlined and people get to quickly pick up what to work on.

**Comment by arctic-hen7** [comment]
Shall do!

**Pull request merged**